emsdk.py

```diff
@@ -665,6 +665,11 @@ def download_file(url, dstpath, download_even_if_exists=False, filename_prefix='
         sys.stdout.flush()
   except Exception as e:
     print("Error downloading URL '" + url + "': " + str(e))
+    # Modified code to handle common "...[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: ..." error
+    # Error results in continuous issues being raised for same problem: #133 #136 #140 #176 #6275 #6548 #6723 #9036  ...
+    r = "<urlopen error \[SSL: CERTIFICATE_VERIFY_FAILED\] certificate verify failed: unable to get local issuer certificate \(_ssl\.c:(\d+)\)>"
+    if re.match(r, str(e)):
+      print("Possible Python SSL issue. Download Python. Install certificates from Python installation folder or command line and try again")
```

Comment:
I think the error message could be a little clearer. Please explain why downloading python can help - I assume because it's a newer version? (so we should suggest downloading the latest one)

Is the sentence with `Install [..]` in addition to the download (a second step), or another option? It's also not clear how to install those, perhaps there's a link we can point to?

Reply:
For prosperity sake,  this is probably the reason the SSL issue is occuring on MacOS. https://bugs.python.org/issue28150#msg276516

Reply:
Updated, see new commit